packages/apps/docs/src/components/InfiniteScroll/InfiniteScroll.tsx

```diff
@@ -47,9 +47,8 @@ export const InfiniteScroll: FC<IProps> = ({
             <Heading as="h5">{error}</Heading>
 
             <Button
-              onClick={() => handleLoad(true)}
-              icon="Refresh"
-              iconAlign="left"
+              onPress={() => handleLoad(true)}
+              startIcon={<SystemIcon.Refresh />}
```

Comment:
if we are using icons again.
Cant we just use <RefreshIcon/> instead of SystemIcon.Refresh.
that can help with treeshaking, no?

Reply:
yes, that's the plan, but didn't want to make a gigantic PR